cms/static/js/views/modals/move_xblock_modal.js

```diff
@@ -30,6 +30,7 @@ function($, Backbone, _, gettext, BaseView, XBlockViewUtils, MoveXBlockUtils, Ht
         options: $.extend({}, BaseModal.prototype.options, {
             modalName: 'move-xblock',
             modalSize: 'lg',
+            modalTop: 110,
```

Comment:
Why are we adding this to just move modal, not the others? I mean we were making move modal same asa other / Edit modal, right ?